### DISCUSSION TITLE
Favoriting/Pinning a specific beatmp sorting option.

### DISCUSSION THREAD

**Discussion by cheeezie** [ppy/osu#22994] [Ideas] [answered]
An option to be able to pin "Sort By" options in the beatmaps screen
![Screenshot 2023-03-27 010802](https://user-images.githubusercontent.com/102518950/227807866-6a580c0c-a6f9-4b0e-a356-8810f99c2d3e.png)

**Comment by Joehuu** [comment, marked as answer]
This won't be possible in the [new design](https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0-1) because it will be changed to just a dropdown.

![firefox_mZnXfQqfhw](https://user-images.githubusercontent.com/35318437/227808277-75dbc0eb-8c54-45de-8fc0-0f090127137c.png)
